rollingpin/main.py

```diff
@@ -188,8 +188,13 @@ def _main(reactor, *raw_args):
         for host in hosts:
             print host.name
     else:
-        deployer = Deployer(config, event_bus, args.parallel,
-                            args.sleeptime, args.timeout)
+        deployer = Deployer(config,
+                            event_bus,
+                            args.parallel,
+                            args.sleeptime,
+                            args.timeout,
+                            args.dangerously_fast,
+                            )
```

Comment:
My reason for leaving the parenthesis on the next line is so future diffs will be much cleaner if we remove/add an argument in the future